patches/ember-cli-addon-docs@6.0.2.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/.vscode/settings.json b/.vscode/settings.json
+new file mode 100644
+index 0000000000000000000000000000000000000000..f6e1d59217c5a9a7b520c5b78336c519ca2360f1
+--- /dev/null
++++ b/.vscode/settings.json
```

Comment:
@AmauryD what is this for? Do we need this file added to ember-cli-addon-docs?

Reply:
There is problem with recent versions of ember-data with ember-cli-addon-docs as mentionned here https://github.com/ember-learn/ember-cli-addon-docs/issues/1507. I made a quick patch until it's fixed.

Reply:
I've just updated to ember-cli-addon-docs v7. No more patch needed ! 🚀